Warning in startup - GenerationTarget encountered exception accepting command : Error executing DDL via JDBC Statement

At startup, the following warning is encountered 

WARN ExceptionHandlerLoggedImpl --- [           main] GenerationTarget encountered exception accepting command : Error executing DDL via JDBC Statement

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL via JDBC Statement
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:67)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:524)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:470)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.createTable(AbstractSchemaMigrator.java:273)
	at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:71)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:203)
	at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:110)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:176)
	at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:65)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:478)
	at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:423)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:711)
	at org.grails.orm.hibernate.cfg.HibernateMappingContextConfiguration.buildSessionFactory(HibernateMappingContextConfiguration.java:276)
	at org.grails.orm.hibernate.connections.HibernateConnectionSourceFactory.create(HibernateConnectionSourceFactory.java:86)
	at org.grails.orm.hibernate.connections.AbstractHibernateConnectionSourceFactory.create(AbstractHibernateConnectionSourceFactory.java:39)
	at org.grails.orm.hibernate.connections.AbstractHibernateConnectionSourceFactory.create(AbstractHibernateConnectionSourceFactory.java:23)
	at org.grails.datastore.mapping.core.connections.AbstractConnectionSourceFactory.create(AbstractConnectionSourceFactory.java:64)
	at org.grails.datastore.mapping.core.connections.AbstractConnectionSourceFactory.create(AbstractConnectionSourceFactory.java:52)
	at org.grails.datastore.mapping.core.connections.ConnectionSourcesInitializer.create(ConnectionSourcesInitializer.groovy:24)
	at org.grails.orm.hibernate.HibernateDatastore.<init>(HibernateDatastore.java:201)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:142)
	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:122)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:271)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1197)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:511)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveConstructorArguments(ConstructorResolver.java:648)
	at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:145)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1197)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.getSingletonFactoryBeanForTypeCheck(AbstractAutowireCapableBeanFactory.java:928)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.getTypeForFactoryBean(AbstractAutowireCapableBeanFactory.java:805)
	at org.springframework.beans.factory.support.AbstractBeanFactory.isTypeMatch(AbstractBeanFactory.java:573)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doGetBeanNamesForType(DefaultListableBeanFactory.java:432)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBeanNamesForType(DefaultListableBeanFactory.java:395)
	at org.springframework.beans.factory.BeanFactoryUtils.beanNamesForTypeIncludingAncestors(BeanFactoryUtils.java:206)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1260)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1094)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1059)
	at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:835)
	at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:741)
	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:467)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1177)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1072)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:511)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.registerBeanPostProcessors(PostProcessorRegistrationDelegate.java:225)
	at org.springframework.context.support.AbstractApplicationContext.registerBeanPostProcessors(AbstractApplicationContext.java:703)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:527)
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:84)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:393)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:380)
	at grails.boot.GrailsApp$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:136)
	at rundeckapp.Application.main(Application.groovy:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: The name "true" is not permitted in this context. Valid expressions are constants, constant expressions, and (in some contexts) variables. Column names are not permitted.
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:259)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1547)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:857)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:757)
	at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7344)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:2713)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:224)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:204)
	at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:734)
	at sun.reflect.GeneratedMethodAccessor131.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
	at com.sun.proxy.$Proxy81.execute(Unknown Source)
	at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:54)
	... 83 common frames omitted


* Rundeck version: 3.1.2-20190927
* install type: war
* OS Name/version: Windows Server 2012
* DB Type/version: mssql

Steps to reproduce the behavior:
1. Restarted rundeck 
java -jar rundeck-3.1.2-20190927.war --skipinstall

REPRODUCED:

```
Configuring Spring Security Core ...
... finished configuring Spring Security Core

[2019-10-02 15:58:36.253]  WARN StandardDialectResolver --- [           main] HHH000385: Unknown Microsoft SQL Server major version [14] using [class org.hibernate.dialect.SQLServer2012Dialect] dialect
Hibernate: create table webhook (id bigint identity not null, version bigint not null, event_plugin varchar(255) not null, plugin_configuration_json varchar(MAX) not null, name varchar(255) not null, enabled bit default true not null, project varchar(255) not null, auth_token varchar(255) not null, primary key (id))
[2019-10-02 15:58:38.618]  WARN ExceptionHandlerLoggedImpl --- [           main] GenerationTarget encountered exception accepting command : Error executing DDL via JDBC Statement

org.hibernate.tool.schema.spi.CommandAcceptanceException: Error executing DDL via JDBC Statement
        at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:67)
        at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlString(AbstractSchemaMigrator.java:524)
        at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.applySqlStrings(AbstractSchemaMigrator.java:470)
        at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.createTable(AbstractSchemaMigrator.java:273)
        at org.hibernate.tool.schema.internal.GroupedSchemaMigratorImpl.performTablesMigration(GroupedSchemaMigratorImpl.java:71)
        at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.performMigration(AbstractSchemaMigrator.java:203)
        at org.hibernate.tool.schema.internal.AbstractSchemaMigrator.doMigration(AbstractSchemaMigrator.java:110)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.performDatabaseAction(SchemaManagementToolCoordinator.java:176)
        at org.hibernate.tool.schema.spi.SchemaManagementToolCoordinator.process(SchemaManagementToolCoordinator.java:65)
        at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:478)
        at org.hibernate.boot.internal.SessionFactoryBuilderImpl.build(SessionFactoryBuilderImpl.java:423)
        at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:711)
        at org.grails.orm.hibernate.cfg.HibernateMappingContextConfiguration.buildSessionFactory(HibernateMappingContextConfiguration.java:276)
        at org.grails.orm.hibernate.connections.HibernateConnectionSourceFactory.create(HibernateConnectionSourceFactory.java:86)
        at org.grails.orm.hibernate.connections.AbstractHibernateConnectionSourceFactory.create(AbstractHibernateConnectionSourceFactory.java:39)
        at org.grails.orm.hibernate.connections.AbstractHibernateConnectionSourceFactory.create(AbstractHibernateConnectionSourceFactory.java:23)
        at org.grails.datastore.mapping.core.connections.AbstractConnectionSourceFactory.create(AbstractConnectionSourceFactory.java:64)
        at org.grails.datastore.mapping.core.connections.AbstractConnectionSourceFactory.create(AbstractConnectionSourceFactory.java:52)
        at org.grails.datastore.mapping.core.connections.ConnectionSourcesInitializer.create(ConnectionSourcesInitializer.groovy:24)
        at org.grails.orm.hibernate.HibernateDatastore.<init>(HibernateDatastore.java:201)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
        at java.lang.reflect.Constructor.newInstance(Unknown Source)
        at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:142)
        at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:122)
        at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:271)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1197)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:511)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
        at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
        at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
        at org.springframework.beans.factory.support.ConstructorResolver.resolveConstructorArguments(ConstructorResolver.java:648)
        at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:145)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.autowireConstructor(AbstractAutowireCapableBeanFactory.java:1197)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1099)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.getSingletonFactoryBeanForTypeCheck(AbstractAutowireCapableBeanFactory.java:928)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.getTypeForFactoryBean(AbstractAutowireCapableBeanFactory.java:805)
        at org.springframework.beans.factory.support.AbstractBeanFactory.isTypeMatch(AbstractBeanFactory.java:573)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.doGetBeanNamesForType(DefaultListableBeanFactory.java:432)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.getBeanNamesForType(DefaultListableBeanFactory.java:395)
        at org.springframework.beans.factory.BeanFactoryUtils.beanNamesForTypeIncludingAncestors(BeanFactoryUtils.java:206)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1260)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1094)
        at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1059)
        at org.springframework.beans.factory.support.ConstructorResolver.resolveAutowiredArgument(ConstructorResolver.java:835)
        at org.springframework.beans.factory.support.ConstructorResolver.createArgumentArray(ConstructorResolver.java:741)
        at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:467)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1177)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1072)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:511)
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
        at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
        at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
        at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
        at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
        at org.springframework.context.support.PostProcessorRegistrationDelegate.registerBeanPostProcessors(PostProcessorRegistrationDelegate.java:225)
        at org.springframework.context.support.AbstractApplicationContext.registerBeanPostProcessors(AbstractApplicationContext.java:703)
        at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:527)
        at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
        at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
        at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
        at grails.boot.GrailsApp.run(GrailsApp.groovy:84)
        at grails.boot.GrailsApp.run(GrailsApp.groovy:393)
        at grails.boot.GrailsApp.run(GrailsApp.groovy:380)
        at grails.boot.GrailsApp$run.call(Unknown Source)
        at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:47)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
        at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:136)
        at rundeckapp.Application.main(Application.groovy:28)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
        at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
Caused by: com.microsoft.sqlserver.jdbc.SQLServerException: The name "true" is not permitted in this context. Valid expressions are constants, constant expressions, and (in some contexts) variables. Column names are not permitted.
        at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDatabaseError(SQLServerException.java:259)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.getNextResult(SQLServerStatement.java:1547)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.doExecuteStatement(SQLServerStatement.java:857)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement$StmtExecCmd.doExecute(SQLServerStatement.java:757)
        at com.microsoft.sqlserver.jdbc.TDSCommand.execute(IOBuffer.java:7344)
        at com.microsoft.sqlserver.jdbc.SQLServerConnection.executeCommand(SQLServerConnection.java:2713)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeCommand(SQLServerStatement.java:224)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.executeStatement(SQLServerStatement.java:204)
        at com.microsoft.sqlserver.jdbc.SQLServerStatement.execute(SQLServerStatement.java:734)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.lang.reflect.Method.invoke(Unknown Source)
        at org.apache.tomcat.jdbc.pool.StatementFacade$StatementProxy.invoke(StatementFacade.java:114)
        at com.sun.proxy.$Proxy81.execute(Unknown Source)
        at org.hibernate.tool.schema.internal.exec.GenerationTargetToDatabase.accept(GenerationTargetToDatabase.java:54)
        ... 83 common frames omitted

[2019-10-02 15:58:49.680]  INFO BootStrap --- [           main] Starting Rundeck 3.1.2-20190927 (2019-09-27) ...
[2019-10-02 15:58:49.680]  INFO BootStrap --- [           main] using rdeck.base config property: C:/Users/Public/rundeck-oss
[2019-10-02 15:58:49.696]  INFO BootStrap --- [           main] loaded configuration: C:\Users\Public\rundeck-oss\etc\framework.properties
[2019-10-02 15:58:49.743]  INFO BootStrap --- [           main] RSS feeds disabled
[2019-10-02 15:58:49.743]  INFO BootStrap --- [           main] Using builtin realm authentication
[2019-10-02 15:58:49.743]  INFO BootStrap --- [           main] Preauthentication is disabled
[2019-10-02 15:58:49.743]  WARN BootStrap --- [           main] Feature 'cleanExecutionHistoryJob' is enabled
Hibernate: select this_.name as y0_ from project this_
[2019-10-02 15:58:49.915]  INFO BootStrap --- [           main] Rundeck is ACTIVE: executions can be run.
[2019-10-02 15:58:49.915]  WARN BootStrap --- [           main] The JVM default encoding is not UTF-8: windows-1252, you may not see output as expected for multibyte locales. Specify -Dfile.encoding=UTF-8 in the JVM options.
Hibernate: select this_.id as y0_ from execution this_ where this_.date_started is not null and this_.date_completed is null and this_.server_nodeuuid is null and this_.date_started<? and (this_.status is null or this_.status<>?)
[2019-10-02 15:58:50.008]  INFO BootStrap --- [           main] Rundeck startup finished in 468ms
Grails application running at http://node01.rundeck.local:4440 in environment: production
Hibernate: select this_.id as id1_3_0_, this_.version as version2_3_0_, this_.date_created as date_cre3_3_0_, this_.uuid as uuid4_3_0_, this_.last_updated as last_upd5_3_0_, this_.[SIZE] as SIZE6_3_0_, this_.record_name as record_n7_3_0_, this_.storage_reference as storage_8_3_0_, this_.file_state as file_sta9_3_0_, this_.file_name as file_na10_3_0_, this_.storage_type as storage11_3_0_, this_.storage_meta as storage12_3_0_, this_.record_type as record_13_3_0_, this_.execution_id as executi14_3_0_, this_.server_nodeuuid as server_15_3_0_, this_.rduser as rduser16_3_0_, this_.job_id as job_id17_3_0_, this_.sha as sha18_3_0_, this_.project as project19_3_0_, this_.expiration_date as expirat20_3_0_ from job_file_record this_ where this_.expiration_date<=? and this_.file_state=? and this_.server_nodeuuid is null
Hibernate: select this_.id as id1_15_0_, this_.version as version2_15_0_, this_.log_output_threshold as log_outp3_15_0_, this_.do_nodedispatch as do_noded4_15_0_, this_.next_execution as next_exe5_15_0_, this_.date_created as date_cre6_15_0_, this_.node_keepgoing as node_kee7_15_0_, this_.node_exclude_os_arch as node_exc8_15_0_, this_.uuid as uuid9_15_0_, this_.node_include as node_in10_15_0_, this_.success_on_empty_node_filter as success11_15_0_, this_.node_exclude_os_version as node_ex12_15_0_, this_.timeout as timeout13_15_0_, this_.node_exclude_precedence as node_ex14_15_0_, this_.node_exclude_name as node_ex15_15_0_, this_.notify_avg_duration_threshold as notify_16_15_0_, this_.day_of_week as day_of_17_15_0_, this_.node_include_os_version as node_in18_15_0_, this_.node_exclude_os_name as node_ex19_15_0_, this_.retry as retry20_15_0_, this_.filter as filter21_15_0_, this_.group_path as group_p22_15_0_, this_.scheduled as schedul23_15_0_, this_.orchestrator_id as orchest24_15_0_, this_.node_threadcount_dynamic as node_th25_15_0_, this_.node_include_name as node_in26_15_0_, this_.multiple_executions as multipl27_15_0_, this_.time_zone as time_zo28_15_0_, this_.rduser as rduser29_15_0_, this_.node_include_os_name as node_in30_15_0_, this_.filter_exclude as filter_31_15_0_, this_.node_exclude as node_ex32_15_0_, this_.node_rank_order_ascending as node_ra33_15_0_, this_.nodes_selected_by_default as nodes_s34_15_0_, this_.node_include_os_arch as node_in35_15_0_, this_.loglevel as logleve36_15_0_, this_.node_exclude_os_family as node_ex37_15_0_, this_.execution_enabled as executi38_15_0_, this_.max_multiple_executions as max_mul39_15_0_, this_.node_include_os_family as node_in40_15_0_, this_.last_updated as last_up41_15_0_, this_.retry_delay as retry_d42_15_0_, this_.workflow_id as workflo43_15_0_, this_.exec_count as exec_co44_15_0_, this_.month as month45_15_0_, this_.hour as hour46_15_0_, this_.log_output_threshold_action as log_out47_15_0_, this_.arg_string as arg_str48_15_0_, this_.user_role_list as user_ro49_15_0_, this_.total_time as total_t50_15_0_, this_.node_rank_attribute as node_ra51_15_0_, this_.server_nodeuuid as server_52_15_0_, this_.default_tab as default53_15_0_, this_.node_exclude_tags as node_ex54_15_0_, this_.seconds as seconds55_15_0_, this_.exclude_filter_uncheck as exclude56_15_0_, this_.ref_exec_count as ref_exe57_15_0_, this_.node_threadcount as node_th58_15_0_, this_.node_include_tags as node_in59_15_0_, this_.job_name as job_nam60_15_0_, this_.schedule_enabled as schedul61_15_0_, this_.year as year62_15_0_, this_.day_of_month as day_of_63_15_0_, this_.node_filter_editable as node_fi64_15_0_, this_.log_output_threshold_status as log_out65_15_0_, this_.description as descrip66_15_0_, this_.minute as minute67_15_0_, this_.project as project68_15_0_ from scheduled_execution this_ where this_.scheduled=?
Hibernate: select this_.id as id1_4_1_, this_.version as version2_4_1_, this_.plugin_name as plugin_n3_4_1_, this_.date_created as date_cre4_4_1_, this_.filetype as filetype5_4_1_, this_.last_updated as last_upd6_4_1_, this_.completed as complete7_4_1_, this_.execution_id as executio8_4_1_, execution_1_.id as id1_2_0_, execution_1_.version as version2_2_0_, execution_1_.scheduled_execution_id as schedule3_2_0_, execution_1_.do_nodedispatch as do_noded4_2_0_, execution_1_.node_exclude_os_arch as node_exc5_2_0_, execution_1_.node_keepgoing as node_kee6_2_0_, execution_1_.succeeded_node_list as succeede7_2_0_, execution_1_.retry_attempt as retry_at8_2_0_, execution_1_.node_include as node_inc9_2_0_, execution_1_.success_on_empty_node_filter as success10_2_0_, execution_1_.node_exclude_os_version as node_ex11_2_0_, execution_1_.timeout as timeout12_2_0_, execution_1_.node_exclude_precedence as node_ex13_2_0_, execution_1_.node_exclude_name as node_ex14_2_0_, execution_1_.node_include_os_version as node_in15_2_0_, execution_1_.node_exclude_os_name as node_ex16_2_0_, execution_1_.retry as retry17_2_0_, execution_1_.filter as filter18_2_0_, execution_1_.orchestrator_id as orchest19_2_0_, execution_1_.node_include_name as node_in20_2_0_, execution_1_.rduser as rduser21_2_0_, execution_1_.retry_original_id as retry_o22_2_0_, execution_1_.execution_type as executi23_2_0_, execution_1_.node_include_os_name as node_in24_2_0_, execution_1_.abortedby as aborted25_2_0_, execution_1_.filter_exclude as filter_26_2_0_, execution_1_.node_exclude as node_ex27_2_0_, execution_1_.node_rank_order_ascending as node_ra28_2_0_, execution_1_.node_include_os_arch as node_in29_2_0_, execution_1_.loglevel as logleve30_2_0_, execution_1_.node_exclude_os_family as node_ex31_2_0_, execution_1_.node_include_os_family as node_in32_2_0_, execution_1_.cancelled as cancell33_2_0_, execution_1_.retry_delay as retry_d34_2_0_, execution_1_.workflow_id as workflo35_2_0_, execution_1_.timed_out as timed_o36_2_0_, execution_1_.failed_node_list as failed_37_2_0_, execution_1_.arg_string as arg_str38_2_0_, execution_1_.user_role_list as user_ro39_2_0_, execution_1_.node_rank_attribute as node_ra40_2_0_, execution_1_.date_completed as date_co41_2_0_, execution_1_.outputfilepath as outputf42_2_0_, execution_1_.server_nodeuuid as server_43_2_0_, execution_1_.will_retry as will_re44_2_0_, execution_1_.retry_execution_id as retry_e45_2_0_, execution_1_.node_exclude_tags as node_ex46_2_0_, execution_1_.exclude_filter_uncheck as exclude47_2_0_, execution_1_.node_threadcount as node_th48_2_0_, execution_1_.node_include_tags as node_in49_2_0_, execution_1_.date_started as date_st50_2_0_, execution_1_.status as status51_2_0_, execution_1_.node_filter_editable as node_fi52_2_0_, execution_1_.project as project53_2_0_ from log_file_storage_request this_ inner join execution execution_1_ on this_.execution_id=execution_1_.id where this_.completed=? and (execution_1_.server_nodeuuid is null)
Hibernate: select this_.id as id1_2_0_, this_.version as version2_2_0_, this_.scheduled_execution_id as schedule3_2_0_, this_.do_nodedispatch as do_noded4_2_0_, this_.node_exclude_os_arch as node_exc5_2_0_, this_.node_keepgoing as node_kee6_2_0_, this_.succeeded_node_list as succeede7_2_0_, this_.retry_attempt as retry_at8_2_0_, this_.node_include as node_inc9_2_0_, this_.success_on_empty_node_filter as success10_2_0_, this_.node_exclude_os_version as node_ex11_2_0_, this_.timeout as timeout12_2_0_, this_.node_exclude_precedence as node_ex13_2_0_, this_.node_exclude_name as node_ex14_2_0_, this_.node_include_os_version as node_in15_2_0_, this_.node_exclude_os_name as node_ex16_2_0_, this_.retry as retry17_2_0_, this_.filter as filter18_2_0_, this_.orchestrator_id as orchest19_2_0_, this_.node_include_name as node_in20_2_0_, this_.rduser as rduser21_2_0_, this_.retry_original_id as retry_o22_2_0_, this_.execution_type as executi23_2_0_, this_.node_include_os_name as node_in24_2_0_, this_.abortedby as aborted25_2_0_, this_.filter_exclude as filter_26_2_0_, this_.node_exclude as node_ex27_2_0_, this_.node_rank_order_ascending as node_ra28_2_0_, this_.node_include_os_arch as node_in29_2_0_, this_.loglevel as logleve30_2_0_, this_.node_exclude_os_family as node_ex31_2_0_, this_.node_include_os_family as node_in32_2_0_, this_.cancelled as cancell33_2_0_, this_.retry_delay as retry_d34_2_0_, this_.workflow_id as workflo35_2_0_, this_.timed_out as timed_o36_2_0_, this_.failed_node_list as failed_37_2_0_, this_.arg_string as arg_str38_2_0_, this_.user_role_list as user_ro39_2_0_, this_.node_rank_attribute as node_ra40_2_0_, this_.date_completed as date_co41_2_0_, this_.outputfilepath as outputf42_2_0_, this_.server_nodeuuid as server_43_2_0_, this_.will_retry as will_re44_2_0_, this_.retry_execution_id as retry_e45_2_0_, this_.node_exclude_tags as node_ex46_2_0_, this_.exclude_filter_uncheck as exclude47_2_0_, this_.node_threadcount as node_th48_2_0_, this_.node_include_tags as node_in49_2_0_, this_.date_started as date_st50_2_0_, this_.status as status51_2_0_, this_.node_filter_editable as node_fi52_2_0_, this_.project as project53_2_0_ from execution this_ where this_.status=?
[2019-10-02 15:58:56.790]  INFO BootStrap --- [      Thread-14] Rundeck Shutdown detected
```

--- a/rundeckapp/webhooks/grails-app/domain/webhooks/Webhook.groovy
+++ b/rundeckapp/webhooks/grails-app/domain/webhooks/Webhook.groovy
@@ -33,6 +33,5 @@ class Webhook {
 
     static mapping = {
         pluginConfigurationJson type: 'text'
-        enabled defaultValue: "true"
     }
 }

